apps/heft/src/schemas/heft.schema.json

```diff
@@ -65,6 +65,83 @@
                     }
                   }
                 }
+              },
+              // Copy Files
+              {
+                "required": ["copyOperations"],
+                "properties": {
+                  "actionKind": {
+                    "type": "string",
+                    "enum": ["copyFiles"]
+                  },
+
+                  "heftEvent": {
+                    "type": "string",
+                    "enum": ["pre-compile", "compile", "bundle", "post-build"]
+                  },
```

Comment:
It would be useful to extract this into a shared resource. Take a look at [`command-line.schema.json`](https://github.com/microsoft/rushstack/blob/master/apps/rush-lib/src/schemas/command-line.schema.json) for examples.

Reply:
I didn't do this, specifically because we purposely omit "clean" from that list, and according to json-schema spec, you cannot override a `$ref` definition :(

Reply:
`Any members other than "$ref" in a JSON Reference object SHALL be ignored.`
https://tools.ietf.org/html/draft-pbryan-zyp-json-ref-03#section-3